usr/lib/common/template.c

```diff
@@ -701,7 +699,7 @@ CK_RV template_unflatten_withSize(TEMPLATE **new_tmpl, CK_BYTE *buf,
         if (long_len == 4)
```

Comment:
that's ugly, mark it with /* NEEDS REWORK */ or similar
or find out what exactly happens here

Reply:
It tries to determine at runtime if we are running on a 32 bit (4byte long type) or 64 bit (8byte long type) platform. At least for z this works. We do it at more places, you can open an issue for that if you want, but i thunk we can remove 32 bit support anyway soon, together with the old openssl code. We probably should do both for the next release.